web/cobrands/greenwich/base.scss

```diff
@@ -15,38 +25,74 @@
 }
 
 #nav-link {
-  background: url('/cobrands/greenwich/tab-red.png') 0 0 no-repeat;
+    background: url("/cobrands/greenwich/tab-red.png") 0 0 no-repeat;
+    top: 0;
+    transform: translateY(-0%);
+}
+
+.main-menu-container {
+    background: $greenwich_dark_grey;
+    color: $contrast_text;
+    padding: 5px 0;
+    margin-top: 10px;
 }
 
+body.mappage, body.authpage, body.twothirdswidthpage, body.alertpage {
+    .page-header-container, .fms-menu-container {
+        display: none;
```

Comment:
This leaves an empty `.main-menu-container` in mobile view, which looks a bit odd to me?

![Screen Shot 2020-12-04 at 09 00 51](https://user-images.githubusercontent.com/739624/101143870-6dd4bd00-360f-11eb-922d-469e2c55c601.png)